index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>My second assignment</title>
+  <link rel="stylesheet" href="styles/styles.css">
+  <link rel="stylesheet" href="styles/reset.css">
+</head>
+
+<body>
+
+<div class="container">
+  <!-- Header Section -->
+  <header class="main-header">
+    <div class="logo">
+    </div>
+
+    <nav class="main-nav">
+    </nav>
+  </header>
+
+  <!-- Hero Section -->
+  <section class="hero">
+  </section>
+
+  <!-- Info Section -->
+
+
+  <section class="site-info1 cf">
+
+      <div class="info-01 columns">
+      </div>
+      <div class="info-02 columns">
+      </div>
+      <div class="info-03 columns">
+      </div>
+      <div class="info-04 columns">
+      </div>
+
+  </section>
+
+  <section class="site-info2 cf">
+
+      <div class="info-05 columns">
```

Comment:
Because these `.columns` occur inside of the `.site-info2` section, there's no need to increment the `.info-` class numbers. Essentially, your HTML for the `.site-info2` section would look just like `.site-info1`, see below:

 ```html
<section class="site-info2 cf">
  <div class="info-01 columns"></div>
  <div class="info-02 columns"></div>
  <div class="info-03 columns"></div>
  <div class="info-04 columns"></div>
</section>
```

Then in your CSS, if you want to style the first column in your `.site-info2` section, you can do the following:

```css
.site-info2 .info-01 {
  background-color: yellow;
}
```